### PR TITLE
PLAT-132840: Show back button on Header

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
back button on Header is shown always on 2022 UX 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
* Cherry-pick
https://github.com/enactjs/sandstone/pull/782
https://github.com/enactjs/sandstone/pull/869
* Remove unused handler
* Remove animation

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-132840

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Added
 
+- `sandstone/Panels.Header` to always show back button
```
Comment (MikyungKim) [inline CHANGELOG.md:9]: Shouldn't it be under `Changed`?

Comment (jeonghee27) [reply CHANGELOG.md:9]: It was my mistake

File: Panels/Header.js [hunk 2]
```diff
@@ -397,35 +338,12 @@ const HeaderBase = kind({
...
-	},
-
 	computed: {
-		className: ({backButtonAvailable, hover, noBackButton, entering, centered, children, type, styler}) => styler.append(
+		className: ({backButtonAvailable, noBackButton, centered, children, type, styler}) => styler.append(
 			{
 				centered,
 				// This likely doesn't need to be as verbose as it is, with the first 2 conditionals
-				showBack: (backButtonAvailable && !noBackButton && (hover || entering)),
+				showBack: (backButtonAvailable && !noBackButton),
```
Comment (MikyungKim) [inline Panels/Header.js:346]: I think this class not needed anymore.

File: Panels/Header.js [hunk 3]
```diff
@@ -18,42 +14,14 @@ import {Children, useContext, useState} from 'react';
...
-		// when in pointer mode and back button is visible but not focused, if 5-way would not focus
-		// the back button, it should hide the back button
-		return !isBackButton({target});
-	},
-	forwardHideBack
-);
-
 /**
  * A header component for a Panel with a `title` and `subtitle`, supporting several configurable
  * [`slots`]{@link ui/Slottable.Slottable} for components.
```
Comment (MikyungKim) [inline Panels/Header.js:27]: Please remove lines 63 to 64.

**Comment by MikyungKim** [comment]
Please update your branch with recent develop.

File: Panels/Header.js [hunk 4]
```diff
@@ -174,17 +139,6 @@ const HeaderBase = kind({
 		 */
 		entering: PropTypes.bool,
```
Comment (MikyungKim) [inline Panels/Header.js:140]: Do we still need this?

**Review by MikyungKim — APPROVED** [review]
:lgtm: :sogood:

**Review by seunghoh — APPROVED** [review]
:lgtm:

**Pull request merged**